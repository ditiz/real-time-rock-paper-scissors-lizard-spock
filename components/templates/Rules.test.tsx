import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import RulesTemplate from "./Rules"

it("renders RulesTemplate unchanged", () => {
  const { container } = render(<RulesTemplate />)
  expect(container).toMatchSnapshot()
})
