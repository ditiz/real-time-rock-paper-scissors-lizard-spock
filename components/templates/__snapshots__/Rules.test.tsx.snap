// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders RulesTemplate unchanged 1`] = `
<div>
  <nav
    class="navbar"
  >
    <div
      class="navbar__logo"
    >
      <img
        alt="logo"
        data-nimg="1"
        decoding="async"
        height="25"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
        style="color: transparent;"
        width="25"
      />
    </div>
    <div
      class="navbar__links"
    >
      <a
        href="/"
      >
        Home
      </a>
      <a
        href="/Rules"
      >
        Rules
      </a>
      <a
        href="/Classic"
      >
        Classic
      </a>
    </div>
  </nav>
  <article
    class="rules"
  >
    <p
      class="text"
    >
      The game is an expansion on the game Rock, Paper, Scissors. Each player picks a variable and reveals it at the same time. The winner is the one who defeats the others. In a tie, the process is repeated until a winner is found.
    </p>
    <section
      class="rules__details"
    >
      <div>
        <img
          alt="rules"
          data-nimg="1"
          decoding="async"
          height="500"
          loading="lazy"
          src="/_next/image?url=%2Fimg.jpg&w=1080&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1080&q=75 2x"
          style="color: transparent;"
          width="500"
        />
      </div>
      <div>
        <p
          class="text"
        >
          Scissors cuts Paper
        </p>
        <p
          class="text"
        >
          Paper covers Rock
        </p>
        <p
          class="text"
        >
          Rock crushes Lizard
        </p>
        <p
          class="text"
        >
          Lizard poisons Spock
        </p>
        <p
          class="text"
        >
          Spock smashes Scissors
        </p>
        <p
          class="text"
        >
          Scissors decapitates Lizard
        </p>
        <p
          class="text"
        >
          Lizard eats Paper
        </p>
        <p
          class="text"
        >
          Paper disproves Spock
        </p>
        <p
          class="text"
        >
          Spock vaporizes Rock
        </p>
        <p
          class="text"
        >
          (and as it always has) Rock crushes Scissors
        </p>
      </div>
    </section>
  </article>
</div>
`;
